### PR TITLE
Baseline checker content

### PR DESCRIPTION
**Pull request by huwd**
Trello: https://trello.com/c/kYj73KEh/433-deploy-baseline-checker

Updates the brexit checker results page, emails and questions (detail components) to use baseline content.

### CODE REVIEW

File: app/views/brexit_checker/email_signup.html.erb [hunk 1]
```diff
@@ -28,15 +28,12 @@
   <main class="govuk-main-wrapper" role="mail">
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds">
-        <h1 class="govuk-heading-l">Get email alerts</h1>
-        <p class="govuk-body">You are signing up for email alerts about: how to prepare for a no deal Brexit.
-        <p class="govuk-body">
-          This will include: Information about Brexit including what you and your business can do to prepare.
```
Comment (huwd) [inline app/views/brexit_checker/email_signup.html.erb]: Moved this to the itnernationalisation YAML for consistency.
hard coded content makes me sad.

File: config/locales/en/brexit_checker/action_audiences.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
 en:
   brexit_checker:
     action_audiences:
-      no_results: This is a safe and secure service. We don't store any of your personal information.
```
Comment (vanitabarrett) [inline config/locales/en/brexit_checker/action_audiences.yml]: Is it intentional that we've removed this messaging around storing of personal information?

File: config/locales/en/brexit_checker/results.yml [hunk 3]
```diff
@@ -1,13 +1,12 @@
 en:
   brexit_checker:
     results:
-      title: "Get ready for a no-deal Brexit: Your results"
```
Comment (vanitabarrett) [inline config/locales/en/brexit_checker/results.yml]: This confused me - I think we remove the title in this commit and it back somewhere else?

File: config/locales/en/brexit_checker/email_signup.yml [hunk 4]
```diff
@@ -2,4 +2,8 @@ en:
   brexit_checker:
     email_signup:
       title: "Brexit: check what you need to do if there is no deal"
-      sign_up: Subscribe
+      sign_up_heading: "Get email alerts"
+      sign_up_message: |
+        <p class="govuk-body">You are signing up for email updates about: how to prepare for new rules from 1 January 2021.</p>
+        <p class="govuk-body">This will include: Information about the transition period including what you, your family, or your business can do to prepare for new rules.</p>
```
Comment (vanitabarrett) [inline config/locales/en/brexit_checker/email_signup.yml]: Should "Information" be capitalised?

File: config/locales/en/brexit_checker/results.yml [hunk 5]
```diff
@@ -8,7 +8,8 @@ en:
       description_no_actions: |
         Based on your responses, you do not need to take any action to prepare for the new rules from 1 January 2021.
       description_no_answers: |
-        To change your answers, follow the link below. This is a safe and secure service. We do not store any of your personal information.
+        To change your answers, follow the link below.
```
Comment (vanitabarrett) [inline config/locales/en/brexit_checker/results.yml]: This won't keep the line breaks:

Screenshot 2020-01-30 at 15 41 23

File: config/locales/en/brexit_checker/action_audiences.yml (hunk 2 above)
Comment (huwd) [reply config/locales/en/brexit_checker/action_audiences.yml]: Yeah I believe so, the doc said replace this where the results are shown. Will confirm with content though - i agree its weird.

File: config/locales/en/brexit_checker/email_signup.yml (hunk 4 above)
Comment (huwd) [reply config/locales/en/brexit_checker/email_signup.yml]: checked with content, amended

File: config/routes.rb [hunk 6]
```diff
@@ -19,11 +19,6 @@
   end
 
   # Routes for the for Brexit Checker
```
Comment (laurentqro) [inline config/routes.rb]: Minor: maybe update comment to mention transition checker

**Pull request merged**